.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,74 @@
+name: Bug report
+description: Create a report describing unexpected or incorrect behavior in astropy.
+labels: Bug
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Thanks for taking the time to fill out this bug report!
+        Please be sure to check out our
+        [contributing guidelines](https://github.com/astropy/astropy/blob/main/CONTRIBUTING.md)
+        and [code of conduct](https://github.com/astropy/astropy/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
GitHub itself links to the code of conduct and contributing guidelines, and new contributors even receive a special prompt to review them, see the screenshot at https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#about-contributing-guidelines. Anyone that chooses to ignore that is just as capable of ignoring any text we might write here.

Reply:
That is just for new contributors though. We also have contributors that pop in rarely and might not remember.

Reply:
The prompt appears for the new contributors or if the guidelines have changed since the last contribution, but the links are there for everyone.